src/frontend/constants.ts

```diff
@@ -1,5 +1,6 @@
 export const EPIC_LOGIN_URL = 'https://legendary.gl/epiclogin'
-export const HYPERPLAY_STORE_URL = 'https://store.hyperplay.xyz?isLauncher=true'
+export const HYPERPLAY_STORE_URL =
+  'https://hyperplay-store-git-feat-libfromstore-hyperplay.vercel.app?isLauncher=true'
```

Comment:
we need https://github.com/HyperPlay-Gaming/hyperplay-store/pull/136 to be merged

Reply:
yeah, added this info to the other PR as well. thanks

Reply:
This is actually just for testing, I will revert these changes

Reply:
reverted